From 4740853c9f18f815024f7a1fd41dda0750f258e3 Mon Sep 17 00:00:00 2001
From: Yury Khrustalev <yury.khrustalev@arm.com>
Date: Thu, 16 Oct 2025 15:54:37 +0100
Subject: [PATCH] Fix comment for VECTOR_BOOL_MODE

gcc/
	* machmode.def (VECTOR_BOOL_MODE): Fix comment.

(cherry picked from commit 4c26e67641cd95dc89fd1d4f771885309fc0a215)
---
 gcc/machmode.def | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/gcc/machmode.def b/gcc/machmode.def
index ccdd2e8b6be3..3045adb85fa5 100644
--- a/gcc/machmode.def
+++ b/gcc/machmode.def
@@ -148,12 +148,12 @@ along with GCC; see the file COPYING3.  If not see
 	mode, with smaller numbers indicating a higher priority.
 
      VECTOR_BOOL_MODE (NAME, COUNT, COMPONENT, BYTESIZE)
-        Create a vector mode called NAME that contains COUNT boolean
-        elements and occupies BYTESIZE bytes in total.  Each boolean
-        element is of COMPONENT type and occupies (COUNT * BITS_PER_UNIT) /
-        BYTESIZE bits, with the element at index 0 occupying the lsb of the
-        first byte in memory.  Only the lowest bit of each element is
-        significant.
+	Create a vector mode called NAME that contains COUNT boolean
+	elements and occupies BYTESIZE bytes in total.  Each boolean
+	element is of COMPONENT type and occupies ((BYTESIZE * BITS_PER_UNIT)
+	/ COUNT) bits, with the element at index 0 occupying the lsb of the
+	first byte in memory.  Only the lowest bit of each element is
+	significant.
 
      OPAQUE_MODE (NAME, BYTESIZE)
         Create an opaque mode called NAME that is BYTESIZE bytes wide.
-- 
2.43.7


